src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Internal/ISnapshotProvider.cs

```diff
@@ -18,6 +18,6 @@ internal interface ISnapshotProvider
     /// </summary>
     /// <returns>An appropriate sample.</returns>
 #pragma warning disable S4049 // Properties should be preferred
-    ResourceUtilizationSnapshot GetSnapshot();
+    Snapshot GetSnapshot();
```

Comment:
The second struct was renamed from `Utilization` to `ResourceUtilization`, while this one was shortened. Why don't we have symmetry?